### PR TITLE
add coin page scaffold

### PR DESCRIPTION
**Pull request by awiermanncasas-bc**
This PR adds the coin page component and the inner layout of the page
with the page header displaying the coin name, code, logo and description
Screen Shot 2022-02-22 at 12 29 16 PM

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/pages/CoinPage/components/PageScaffold/styles.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  justify-content: center;
+`
+
+export const Grid = styled.div`
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  column-gap: 32px;
+  padding: 40px 64px;
+  width: 100%;
+  max-width: 1280px;
```
Comment (awiermanncasas-bc) [inline packages/blockchain-wallet-v4-frontend/src/pages/CoinPage/components/PageScaffold/styles.ts:14]: @blockdylanb  do we have a specification for the max width of a page?
or @ethan-bc should we have this specification in constellation?

File: packages/blockchain-wallet-v4-frontend/src/pages/CoinPage/components/CoinHeader/CoinHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+import { Image, Text } from 'blockchain-info-components'
```
Comment (awiermanncasas-bc) [inline packages/blockchain-wallet-v4-frontend/src/pages/CoinPage/components/CoinHeader/CoinHeader.tsx:3]: @blockdylanb should we use components from the `blockchain-info-components` or should we implement these in constellation too?

Comment (blockdylanb) [reply packages/blockchain-wallet-v4-frontend/src/pages/CoinPage/components/CoinHeader/CoinHeader.tsx:3]: Images still come from info components, so I think what you have here is ok for now. Text could possibly come from constellation but I haven't actually used it yet so would need to see if it's working properly.

File: packages/blockchain-wallet-v4-frontend/src/pages/CoinPage/components/PageScaffold/styles.ts (hunk 1 above)
Comment (blockdylanb) [reply packages/blockchain-wallet-v4-frontend/src/pages/CoinPage/components/PageScaffold/styles.ts:14]: Ethan mentioned having a max width in our meeting but I don't think he had decided on an actual number at that point. Let's go with 1280 for now and adjust as needed. 👍🏻

**Review by blockdylanb — APPROVED** [review]

File: packages/blockchain-wallet-v4-frontend/src/pages/CoinPage/components/CoinHeader/CoinHeader.tsx (hunk 2 above)
Comment (awiermanncasas-bc) [reply packages/blockchain-wallet-v4-frontend/src/pages/CoinPage/components/CoinHeader/CoinHeader.tsx:3]: yes, I created a spike PR for the text in constellation but I think we still need to wait for Pedro to have the theme done and then they will revisit the Text component in constellation.

So I think there is a long way until we can get the Text component from constellalation.

Should I leave like this for now?

What I could do is try to revisit  the Text component in the blockchain info component to make it type safe, what you think?

Comment (blockdylanb) [reply packages/blockchain-wallet-v4-frontend/src/pages/CoinPage/components/CoinHeader/CoinHeader.tsx:3]: @awiermanncasas-bc Given that it'll go away eventually I think you can just leave it as is for now.

**Pull request merged**